cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -715,7 +715,8 @@ describe("Timeline", () => {
             cy.get(".mx_RoomHeader").findByRole("button", { name: "Search" }).click();
 
             cy.get(".mx_SearchBar").percySnapshotElement("Search bar on the timeline", {
-                widths: [320, 640], // Emulate narrow timeline (cf. decryption-failure.spec.ts)
+                // Emulate narrow timeline (apply the same values used on decryption-failure.spec.ts)
```

Comment:
There is a danger here that  decryption-failure.spec.ts will be changed without this file being updated, making the comment very confusing.

I'm also wondering if it actually matters that we use the same value in both places; I think it does not matter. So, this comment adds no value. I suggest we remove it.

```suggestion
                // Emulate narrow timeline
```

Reply:
That makes sense. Ideally we should use variables here, but I do not have knowledge about it unfortunately…